server/src/main/java/com/paypal/selion/utils/ConfigParser.java

```diff
@@ -48,6 +48,10 @@
      *         represented by the JSON file passed via the JVM argument <b>SeLionConfig</b>
      */
     public static ConfigParser parse() {
+        LOGGER.entering();
+        if (parser == null) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Singleton is not thread safe](https://www.codacy.com/app/SeLion/paypal-SeLion/file/1516698859/issues/source?bid=2928425&fileBranchId=3008049#l52)